src/applications/accreditation/21a/tests/21a.cypress.spec.js

```diff
@@ -66,7 +64,7 @@ const testConfig = createTestConfig(
         [
           { text: 'Have you ever served in the military?', value: true },
           {
-            text: 'Do you have another military service experience?',
+            text: 'Do you have another military service experience to add?',
```

Comment:
The wording of this question seems off. `Do you have any other military service experience...` sounds better to me. Is this what's specified in the designs?